h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -73,11 +78,56 @@ class SearchBarController extends Controller {
     };
 
     var openDropdown = () => {
+      clearActiveDropdownItem();
+
+      // Reset dropdown items to not hidden
+      Object.keys(this._dropdownItems).forEach(function(key) {
+        this._dropdownItems[key].classList.remove('is-hidden');
+      }.bind(this));
```

Comment:
Is it possible to write this more simply? Does `this_.dropdownItems.forEach(function(dropdownItem) {dropdownItem.classList.remove('is-hidden')})` not work?